src/lightning/pytorch/loggers/neptune.py

```diff
@@ -223,8 +223,8 @@ def __init__(
         prefix: str = "training",
         **neptune_run_kwargs: Any,
     ):
-        if not _NEPTUNE_AVAILABLE and not _NEPTUNE_CLIENT_AVAILABLE:
-            raise ModuleNotFoundError(str(_NEPTUNE_AVAILABLE))
+        if not _NEPTUNE_AVAILABLE and not _LEGACY_NEPTUNE_CLIENT:
+            raise ModuleNotFoundError("`neptune` package is required. Run `pip install neptune`")
```

Comment:
The previous code was already printing this hint and additional info when the package was not found.